app/models/educator.rb

```diff
@@ -17,14 +17,14 @@ class Educator < ActiveRecord::Base
 
   validates :email, presence: true, uniqueness: true
 
-  validate :has_school_unless_districtwide,
-           :admin_gets_access_to_all_students,
-           :grade_level_access_is_array_of_strings,
-           :grade_level_strings_are_valid,
-           :grade_level_strings_are_uniq,
-           :grade_level_access_is_not_nil
+  validate :validate_has_school_unless_districtwide,
+           :validate_admin_gets_access_to_all_students,
+           :validate_grade_level_access_is_array_of_strings,
+           :validate_grade_level_strings_are_valid,
+           :validate_grade_level_strings_are_uniq,
+           :validate_grade_level_access_is_not_nil
 
-  VALID_GRADES = [ 'PK', 'KF', '1', '2', '3', '4', '5', '6', '7', '8' ].freeze
+  VALID_GRADES = [ 'PK', 'KF', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12' ].freeze
```

Comment:
`TestPals` revealed that this was a bug and wouldn't allow grade-level access at the high school.

Reply:
Ayyyy great catch. 👍